app/views/dossiers/_attestation.html.haml

```diff
@@ -0,0 +1,14 @@
+- if dossier.attestation.present?
+  #attestation-recapitulatif.default-data-block
+    .row.show-block.clearfix
+      .header
+        .title
+          .carret-right
+          .carret-down
+          ATTESTATION
+      .details
+        .left
+          = image_tag('pdf.svg', width: '20px')
```

Comment:
Alors historiquement ce n'est pas sale et cela permet(tait) d'éviter les reflow : https://www.voorhoede.nl/en/blog/say-no-to-image-reflow/
https://www.codecaptain.io/blog/web-development/responsive-images-and-preventing-page-reflow/474

Mais je ne retrouve plus la réference dans google developer. Donc peut-être est-ce dorénavant à mitiger.

Veux tu qd mm que je déplace le width ?